src/site/content/en/reliable/codelab-background-sync/index.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: codelab
+title: Building a resilient search experience with Workbox
+authors:
+  - demianrenzulli
+  - jeffposnick
+date: 2020-06-17
+description: |
+  How to implement a resilient search experience with Workbox, using  Background Sync and the Push API.
+glitch: resilient-search-codelab
+---
+
+This codelab shows you how to implement a resilient search experience with Workbox. The demo app it uses contains a search box that calls a server endpoint, and redirects the user to a basic HTML page.
+
+{% Aside %} 
+This codelab uses [Chrome DevTools](https://www.google.com/chrome/). Download Chrome if you don't already have it. 
+{% endAside %}
+
+## Measure
+
+Before adding optimizations, it's always a good idea to first analyze the current state of the application.
+
+{% Instruction 'remix' %}
+{% Instruction 'preview' %}
+
+Check how the website behaves when going offline:
```

Comment:
```suggestion
In the new tab that just opened, check how the website behaves when going offline:
```
I want to make it clearer that they should be working in the new tab, not the embedded preview